autogen/retrieve_utils.py

```diff
@@ -254,6 +254,7 @@ def get_files_from_dir(dir_path: Union[str, List[str]], types: list = TEXT_FORMA
 def get_file_from_url(url: str, save_path: str = None):
     """Download a file from a URL."""
     if save_path is None:
+        os.makedirs("/tmp/chromadb", exist_ok=True)
         save_path = os.path.join("/tmp/chromadb", os.path.basename(url))
```

Comment:
Shouldn't **save_path = os.path.join("/tmp/chromadb", os.path.basename(url))** be outside the **if block**?
I guess the login here is you are not checking if the tmp path exists or not. Looks good!

Reply:
Updated. Checked both save_path is given or not. If not, i.e., None, use /tmp/chromadb as the download folder; else make sure the folder is created.

Reply:
Check out this issue https://github.com/microsoft/autogen/issues/444

@pcdeadeasy @thinkall